### DISCUSSION TITLE
Backend.ai 관련 구조를 설명하는 Youtube, 슬라이드 모음입니다.

### DISCUSSION THREAD

**Discussion by mindgitrwx** [lablup/contribution-academy-2021#102] [General]
## Oct, 2018
[보다 싸고 빠르고 안전한 AI 개발의 미래 - 김준기님](https://www.sosconhistory.net/soscon2018/pdf/day2_1630_1.pdf)

[Callosum: An RPC Tranport Library - 김준기님](https://www.youtube.com/watch?v=3V4-9JfQEvg)
[slide](https://speakerdeck.com/achimnol/pycon-kr-2018-callosum-an-rpc-transport-library)

## Aug, 2019
[머신러닝 및 데이터 과학 연구자를 위한 python 기반 컨테이너 분산처리 플랫폼 설계 및 개발 - PyCon.KR 2019 - 신정규님](https://www.youtube.com/watch?v=dPVHgFGMw-c&ab_channel=PyConKorea)
[slide](https://www.slideshare.net/inureyes/python-165125392)

## Sep, 2019
[Real-world asyncio - PyCon.KR 2019 - 김준기님](https://www.youtube.com/watch?v=QaiczQzJAmA)
[slide](https://speakerdeck.com/achimnol/pycon-kr-2019-real-world-asyncio)

## Nov, 2020
[Backend.AI에 점진적 typing 적용하기 - PyCon Korea - 김준기님](https://www.youtube.com/watch?v=9njdWZaTZYs)

[Backend.AI의 오픈소스 문서 작성과 자동화 경험 - 신정규님](https://www.youtube.com/watch?v=Scj3YCVPsAU)
[pdf](https://kcd-2020.s3.ap-northeast-2.amazonaws.com/keynotes/%E1%84%89%E1%85%B5%E1%86%AB%E1%84%8C%E1%85%A5%E1%86%BC%E1%84%80%E1%85%B2_KCD2020_-_Documenting_BackendAI.pdf)

**Comment by lizable** [comment]
Awesome! Probably we could add presentations keynotes shared in the code-base seminar in this thread, too.
